tests/pipeline/test_blaze.py

```diff
@@ -1128,3 +1131,82 @@ def test_novel_deltas_macro(self):
                 window_length=3,
                 compute_fn=op.itemgetter(-1),
             )
+
+    def test_ignore_nonexistent_delta(self):
```

Comment:
This test has a structure very similar to `test_novel_deltas`. We could simplify it a bit, figure out a way to parameterize (even though there's an extra context manager here, so that might get tricky), or just leave it as is.